runbooks/how-to-update-for-rda-api-changes.md

```diff
@@ -0,0 +1,53 @@
+        - Migrations: https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-model/bfd-model-rif/src/main/resources/db/migration
+        - The [bfd-model-rda](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-model/bfd-model-rda) module can be built to auto-generate a full sql script to aid in migration creation
+   5. Run [bfd-db-migrator](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-db-migrator) tests to ensure validity of migration
+3. Update [bfd-pipeline-rda-grpc](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc) module source code
+   1. Update [RdaService.java](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RdaService.java) with with new RDA_PROTO_VERSION
+   2. Update the Random*ClaimGenerators for local testing with new columns/entities
+   3. Update tests with new data fields, for example:
+      - [FissClaimRdaSinkIT](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSinkIT.java)
+      - [FissClaimStreamCallerIT](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimStreamCallerIT.java)
+      - [FissClaimTransformerTest](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformerTest.java)
+      - [StandardGrpcRdaSourceIT](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSourceIT.java)
+      - etc.
+4. Create FHIR mappings for new RDA fields
+   1. Pull new fields from RDA data dictionary
+   2. Reach out to FHIR SME to research new fields and identify appropriate FHIR mappings
+   3. Vet new mappings for what has value to customers, not all fields may get mapped
+   4. Create / update Claim / ClaimResponse model structure [in confluence](https://confluence.cms.gov/display/BCDA/PACA+FHIR+Resources)
+      - New mapping entries in tables
+      - Update Claim / ClaimResponse sample JSON responses for FISS / MCS claim types.
+5. Update [bfd-pipeline-rda-bridge](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-pipeline/bfd-pipeline-rda-bridge) module source code
+   1. Identify feasible synthetic CCW data values for corresponding new RDA fields
+   2. Update transformer with new field mappings
+   3. Update tests with new expectations
+      - May need to update the test RIF (csv) files with more/better data for new data fields
+6. Deploy SBX update
+   1. Create new synthetic RDA data using the synthetic CCW data from synthea
```

Comment:
I feel like Maybe `Deploy SBX update.` is 6 with no bullets and the rest are `7. Update Sandbox S3 bucket with new synthetic data`.  We need the code to be in place on sandbox before we update the S3 bucket so that the server doesn't get confused with new JSON and old code.